transformations/multilingual_lexicon_perturbation/test.json

```diff
@@ -0,0 +1 @@
, +36 % ) ."}, "outputs": [{"sentence": "RESULTS Repaglinide was also associated with an 增加 在 the AUC ( 60 ) 和 AUC ( 120 ) for insulin ( +56 % , +61 % ) 和 C-peptide ( +41 % , +36 % ) ."}]}, {"class": "MultilingualLexiconPerturbation", "inputs": {"sentence": "1833,worldcom ex-boss launches defence lawyers defending former worldcom chief bernie ebbers against a battery of fraud charges have called a company whistleblower as their first witness."}, "outputs": [{"sentence": "1833 , worldcom ex-boss launches defence lawyers defending former worldcom chief bernie ebbers 反对 a battery 由 fraud charges have called a 公司 whistleblower as their first 见证 ."}]}, {"class": "MultilingualLexiconPerturbation", "inputs": {"sentence": "last week  mci agreed to a buyout by verizon communications in a deal valued at $6.75bn.,business"}, "outputs": [{"sentence": "last week mci agreed 到 a buyout by verizon communications 在 a 交易 valued 在 $ 6.75bn. , business"}]}, {"class": "MultilingualLexiconPerturbation", "inputs": {"sentence": "the sequel took $44.7m (£23.2m) between 24 and 26 december  according to studio estimates"}, "outputs": [{"sentence": "the sequel took $ 44.7m ( £23.2m ) between 24 和 26 december 根据 to 工作室 estimates"}]}, {"class": "MultilingualLexiconPerturbation", "inputs": {"sentence": "australian open champion williams survived an early scare to beat russia s elena bovina 1-6 6-1 6-4. world number one lindsay davenport and anastasia myskina also progressed"}, "outputs": [{"sentence": "australian open champion williams survived an 早点 scare 到 beat russia s elena bovina 1-6 6-1 6-4. 世界 number 一 lindsay davenport 和 anastasia myskina also progressed"}]}, {"class": "MultilingualLexiconPerturbation", "inputs": {"sentence": "odds on kenzie s victory have been slashed from 4/6 to 1/2  although bookmakers have said they are not ruling out a late rally from former happy mondays star bez"}, "outputs": [{"sentence": "odds on kenzie s 胜利 有 been slashed from 4/6 到 1/2 although bookmakers 有 said 他们 are not ruling out a 迟到 rally from former 幸福 mondays star bez"}]}, {"class": "MultilingualLexiconPerturbation", "inputs": {"sentence": "other pre-electronic gadgets in the top 100 include the sextant from 1731 (59th position)  the marine chronometer from 1761 (42nd position) and the kodak brownie camera from 1900 (28th position)"}, "outputs": [{"sentence": "other pre-electronic gadgets in 其它 顶部 100 include 其它 sextant from 1731 ( 59th 立场 ) 其它 marine chronometer from 1761 ( 42nd position ) and 其它 kodak brownie camera from 1900 ( 28th 立场 )"}]}, {"class": "MultilingualLexiconPerturbation", "inputs": {"sentence": "moya fights back for indian title carlos moya became the first man to successfully defend the chennai open title by beating four-times finalist paradorn srichaphan 3-6 6-4 7-6 (7/5)"}, "outputs": [{"sentence": "moya fights back for 印度人 标题 carlos moya became the 首先 man 到 成功的 defend the chennai open 标题 by beating four-times finalist paradorn srichaphan 3-6 6-4 7-6 ( 7/5 )"}]}]}
```

Comment:
Btw, could you please format the json file so that it is not displayed as one line? Thanks!

Reply:
Sure, I have updated it to a more readable format